@kadena/chainweb-stream-client - Changed default export at index to named export

Takadenoshi opened the pull request:
Default export didn't work with out CJS-targeted output

Also broke the example in the README

alber70g commented on common/changes/@kadena/chainweb-stream-client/chore-chainweb-stream-client-change-default-export_2023-07-19-12-19.json:
This one needs to be created with changesets now

Takadenoshi replied:
done

Takadenoshi commented:
This needs publishing, the 0.1.0 version that was published to npm from main overwrote the 0.0.10 version that had this fix

webpro commented:
> This needs publishing, the 0.1.0 version that was published to npm from main overwrote the 0.0.10 version that had this fix

- We should not bump `version` ourselves manually (Changesets does this for us)
- On the same note, we do need to make sure the changelog reflects the bump

I see `minor` in the changelog, but a `patch` bump in `package.json#version`

Takadenoshi commented:
- I reverted the manual bump in package.json
- I changed the changeset bump to a patch

Ideally I wanted to stay on 0.0.x for a while until it is a bit more stable but now we are on 0.1.x
